aspnetcore/blazor/get-started.md

```diff
@@ -102,7 +100,10 @@ In a few steps, get started with Blazor:
    cd WebApplication1
    dotnet run
    ```
-  
+   
+   > [!NOTE]
```

Comment:
Can we move this next to the instructions for calling `dotnet new blazorserverside`?

Reply:
😕 ... there are no more instructions. This is the last tab.

Reply:
Also, should we reverse the order so that `dotnet new blazor` is first?